docs/references/protocol/transactions/types/nftokenmodify.md

```diff
@@ -0,0 +1,39 @@
+---
+seo:
+    description: Modify a dynamic NFT.
+labels:
+  - Non-fungible Tokens, NFTs
+title:
+  - NFTokenModify
+---
+# NFTokenModify
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/xrpld/app/tx/detail/NFTokenMint.cpp "Source")
+
+`NFTokenModify` is used to change the `URI` field of an NFT to point to a different URI in order to update the supporting data for the NFT. The NFT must have been minted with the `tfMutable` flag set. See [Dynamic Non-Fungible Tokens](../../../../concepts/tokens/nfts/dynamic-nfts.md).
+
+## Example {% $frontmatter.seo.title %} JSON
+
+
+```json
+{
+  "TransactionType": "NFTokenModify",
+  "Account": "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
+  "Owner": "rogue5HnPRSszD9CWGSUz8UGHMVwSSKF6",
+  "Fee": "10",
+  "Sequence": 33,
+  “NFTokenID”: “0008C350C182B4F213B82CCFA4C6F59AD76F0AFCFBDF04D5A048C0A300000007",
+  "URI": "697066733A2F2F62616679626569636D6E73347A736F6C686C6976346C746D6E356B697062776373637134616C70736D6C6179696970666B73746B736D3472746B652F5665742E706E67"
+  }
+```
+
+{% raw-partial file="/docs/_snippets/tx-fields-intro.md" /%}
+
+| Field             | JSON Type           | [Internal Type][] | Description        |
+|:------------------|:--------------------|:------------------|:-------------------|
+| `TransactionType` | String              | UINT16            | Type is `NFTokenModify`. |
+| `Account`         | String              | AccountID        |  The unique address of either the issuer or an authorized minter of the NFT. |
+| `Owner`           | String              | AccountID        | Address of the owner of the NFT. |
```

Comment:
If Account==Owner, this field should not be specified, so it is an Optional field.